### DISCUSSION TITLE
Possible minor typos in manual download links of composer download page

### DISCUSSION THREAD

**Discussion by alexandreelise** [composer/composer#10677] [General]
Hello composer team. I think I found multiple minor typos in download links "title" attribute

The links are below the this part of this page https://getcomposer.org/download/

Manual Download
If you prefer to download the phar manually, here are the available versions:
...

At the time of writing this there are extras "s" when you hover the links it shows "lastest" rather than "latest".
It's seems to be minor typos.
Just trying to help as much as I can.

@Seldaek

**Comment by Seldaek** [comment]
Thanks, fixed in https://github.com/composer/getcomposer.org/pull/207

**Reply by alexandreelise** [reply]
You welcome Jordi @Seldaek
